src/admin/dto/full-report.dto.ts

```diff
@@ -26,6 +26,12 @@ export class FullReportDto {
   @ApiProperty()
   email: string;
 
+  @ApiProperty()
+  controle: string;
+
+  @ApiProperty()
+  controleId: string;
```

Comment:
Consider renaming `controle` and `controleId` for clarity and more english souding names.